scripts/zones/Dynamis-Valkurm/MobIDs.lua

```diff
@@ -0,0 +1,13 @@
+-----------------------------------
+-- 
+-- Zone: Dynamis-Valkurm
+-- 
+-----------------------------------
+
+TE5_RANDOM =
+{
+    [0] = 16937500, -- Warchief_Tombstone
```

Comment:
lua arrays start at 1

Reply:
in that case (having the array start at 1) i'd probably just simply delete the indexes and let lua auto assign... yea?

Reply:
it's a lua array, so we will want to follow the convention and make it start at 1, so that we can use ipairs later when we iterate over it

Reply:
yeah, you should be able to do that.  we don't actually care about the index at any point so we can let lua do whatever it wants